Bottom: fe2f557e7ec25b277766a271fbef4dc22e11f002
Top:    68379ba37f5b2e27d9b0df803a3bc56528eaee2a
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:58 -0500

Ignore exim4.conf.template

We don't use this file, so ignore it


---
diff --git a/docker/docker-exim4/docker-entrypoint.sh b/docker/docker-exim4/docker-entrypoint.sh
index 7080bd4..84e1ca0 100755
--- a/docker/docker-exim4/docker-entrypoint.sh
+++ b/docker/docker-exim4/docker-entrypoint.sh
@@ -21,7 +21,6 @@ chown -R root:mail /var/mail
 case ${1} in
     exim4)
         if   [[ -f ${CONFIG_PATH}/update-exim4.conf.conf ]] \
-          && [[ -f ${CONFIG_PATH}/exim4.conf.template ]] \
           && [[ ! -f ${CONFIG_PATH}/exim4.conf ]]
         then
             # set the hostname
